.flake8

```diff
@@ -4,10 +4,11 @@
 # W: style warnings
 # C: complexity
 # E402: module level import not at top of file
+# F821: undefined name 'c' in hub image files
```

Comment:
I think F821 can be really helpful.

I suggest instead we declare the following in the flake8 config on a line below.

```
builtins = c
```

Reply:
Done in [`8543569`](https://github.com/2i2c-org/infrastructure/pull/939/commits/85435698922bb7262a38006f1ab37b8e5c93eff5)